ND_Supporting_Document_3_0.adoc

```diff
@@ -1539,14 +1544,11 @@ Note: Testing the validity of the client certificate is performed as part of X.5
 [arabic, start=311]
 . The evaluator shall perform the following tests:
 [loweralpha, start=1]
-.. Test 1: The evaluator shall perform the following tests for each selected version of TLS:
-[lowerroman]
-... The evaluator shall examine the Certificate Request message. The evaluator shall verify that the SignatureAndHashAlgorithms (TLS 1.2) or the SignatureSchemes (TLS 1.3) match the SignatureAndHashAlgorithms specified in the requirement.
-... The evaluator shall establish a connection and authenticate the client using each selected SignatureAndHashAlgorithm. The evaluator shall verify the connection succeeds.
-.. Test 2 [conditional]: If TLS 1.3 is supported, the evaluator shall perform the following tests if “present the signature_algorithms_cert extension” is selected:
-[lowerroman]
-... The evaluator shall examine the Certificate Request message and verify it contains the signature_algorithms_cert extension and the SignatureSchemes match the SignatureSchemes specified in the requirement.
-... The evaluator shall establish a TLS connection and perform client authentication using a certificate chain using each of the SignatureSchemes specified by the requirement. The evaluator shall ensure the signatures used in the certificate chain are consistent with the SignatureScheme being tested.
+.. Test 1: The evaluator shall initiate a TLS session from a test TLS client and examine the Certificate Request message sent by the TSF. The evaluator shall verify the Certificate Request message contains the signature_algorithms or signature_algorithm_cert extension. To view the Certificate Request message in TLS 1.3, the message will need to be decrypted.
++
+For TLS 1.2, the evaluator shall verify the SignatureAndHashAlgorithms values match the selections specified in the requirement.
++
+For TLS 1.3, the evaluator shall verify the SignatureSchemes values match the selections specified in the requirement.
```

Comment:
The "b." part of the Test for FCS_TLSS_EXT.2.4/FCS_DTLSS_EXT.2.4 should be added.  It's unclear to me exactly what that should be as I don't know what the author intended here. When the tests were separate, the options were different.  For reference, both are listed below:

The evaluator shall establish a connection and authenticate the client using each selected SignatureAndHashAlgorithm. The evaluator shall verify the connection succeeds.

The evaluator shall establish a DTLS connection and perform client authentication using a certificate chain using each of the SignatureSchemes specified by the requirement. The evaluator shall ensure the signatures used in the certificate chain are consistent with the SignatureScheme being tested.

Reply:
@plughy2 I'm not sure I understand your comment. Are you saying the recommended changes in issue #318 were incomplete?

Reply:
@mclearn @kenji-lightship 
Are either of you able to assist with this?

Reply:
The entire test for 2.4 (both tests 1 and 2) seem to need some rework. I’ve spent some time on that set of requirements for the TLS pkg so I may be able to provide the missing (b) part. I won’t be able to get to it until tomorrow.

Reply:
@mclearn That would be great. THis is our last edit before we plan to publish 3.0e today. APologies for the added pressure. Feel free to add a commit to the TLS-RFI-Issues branch with edits. Thanks!